.github/workflows/jdk-early-access-build.yml

```diff
@@ -0,0 +1,100 @@
+    timeout-minutes: 360
+    env:
+      MAVEN_OPTS: -Xmx2048m -XX:MaxMetaspaceSize=1000m
+    steps:
+
+      - name: Set up JDK
+        # AdoptOpenJDK does not seem to provide EA builds, so use the standard action which provides a Zulu JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JDK_VERSION }}
+      - name: Print Java version
+        shell: bash
+        run: java -version
+
+      - name: Stop mysql
+        shell: bash
+        run: |
+          ss -ln
+          sudo service mysql stop || true
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.ref || 'main' }}
+      - uses: n1hility/cancel-previous-runs@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Reclaim Disk Space
+        run: .github/ci-prerequisites.sh
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "::set-output name=date::$(/bin/date -u "+%Y-%m")"
+        shell: bash
+      - name: Cache Maven Repository
+        id: cache-maven
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          # refresh cache every month to avoid unlimited growth
+          # also add JDK version to key to avoid poisioning of the CI cache
+          key: q2maven-${{ steps.get-date.outputs.date }}-${{ env.JDK_VERSION }}
+      - name: Build with Maven
+        # try to gather as many failures as possible, test reports are evaluated in subsequent actions
+        run: |
+          mvn $JVM_TEST_MAVEN_OPTS -Dtcks install -fae -Dmaven.test.failure.ignore=true
+      - name: Publish Test Report
+        if: always()
+        uses: scacap/action-surefire-report@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # fail required to upload failure archive in subsequent actions
+          fail_on_test_failures: true
+      - name: Publish Gradle Test Report
+        if: always()
+        uses: scacap/action-surefire-report@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # fail required to upload failure archive in subsequent actions
+          fail_on_test_failures: true
+          check_name: 'Test Report for Gradle'
+          report_paths: '**/build/test-results/test/TEST-*.xml'
+      - name: Prepare failure archive (if maven failed)
+        if: failure()
+        shell: bash
+        run: find . -name '*-reports' -type d -o -name '*.log' | tar -czf test-reports.tgz -T -
+      - name: Upload failure Archive (if maven failed)
+        uses: actions/upload-artifact@v1
+        if: failure()
+        with:
+          name: test-reports-linux-jvm${{ env.JDK_VERSION }}
+          path: 'test-reports.tgz'
+      - name: Delete Local Artifacts From Cache
+        if: always()
+        shell: bash
+        run: rm -r ~/.m2/repository/io/quarkus
```

Comment:
We need to publish the results to an issue so that we can include it in status.quarkus.io.

You can find an example here: https://github.com/quarkusio/quarkus/blob/main/.github/workflows/deploy-snapshots.yml#L51 .

Reply:
Ah well, I was hoping that someone else can add this part in a follow up PR.
I'll have a short look later and let you know whether I have the capacity for this currently.

Reply:
@gsmet Leaving my previous comment aside for a moment, do you think we should only publish the _scheduled_ runs to the issue?
With the manual trigger you can select an entirely diffent JDK version, so maybe this should not be forwarded to the issue. Or we could check whether the manual run was triggered with the default JDK version and only then report it...

Reply:
#15867 is the tracking issue. Will try first in my fork with another one.

Reply:
I added the reporting, tested in my fork with a custom token an issue: https://github.com/famod/quarkus/issues/75